scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -40,6 +40,17 @@ import org.ossreviewtoolkit.spdx.SpdxConstants
 import org.ossreviewtoolkit.spdx.calculatePackageVerificationCode
 import org.ossreviewtoolkit.utils.textValueOrEmpty
 
+data class LicenseGrouping(
```

Comment:
Commit message:
- License choices and expressions get mixed up. Please only refer to license expressions.
- s/Closes: #3523./Closes #3523./